.github/workflows/monitoring.yml

```
@@ -50,23 +32,29 @@ jobs:
+          cache: pnpm
 
       - name: Install dependencies
         run: pnpm install --frozen-lockfile
 
       - name: Run health checks
         id: health-check
         run: |
-          echo "🏥 Running PRODUCTION health checks against ${{ env.APP_URL }}"
+          echo "🏥 Running PRODUCTION health checks against ${{ vars.APP_URL }}"
           echo "ℹ️  Production failures will trigger alerts and notifications"
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)